### ISSUE TITLE
error while fitting a simple french dataset

### ISSUE THREAD

**Issue by McFoggy** [snipsco/snips-nlu#616]
while training a dedicated dataset in french language, I received the following error

```
thread '<unnamed>' panicked at 'No such local time', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/chrono-0.3.0/src/offset/mod.rs:151:34
note: Run with `RUST_BACKTRACE=1` for a backtrace.
fatal runtime error: failed to initiate panic, error 5
Aborted
root@d829fbc31689:/intents/fr# thread '<unnamed>' panicked at 'No such local time', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/chrono-0.3.0/src/offset/mod.rs:151:34
note: Run with `RUST_BACKTRACE=1` for a backtrace.
fatal runtime error: failed to initiate panic, error 5
```

I have reduced my intent examples to a one line file causing troubles:
```
l'enfant pèse [weight:snips/number](31,2) [weightUnit:weightUnit](kilogrammes)
```

while the `entity_weightUnit.txt` contains 
```
kg,kilogramme,kilogrammes,kilo,kilos
g,gramme,grammes
```

I have seen #563 but my intent does not refer anyhow to a date.

the version in use looks to be 0.15.0
```
root@d829fbc31689:~# cat /usr/local/lib/python3.6/site-packages/snips_nlu/__about__.py | grep version
# https://python-packaging-user-guide.readthedocs.io/guides/single-sourcing-package-version/
__version__ = "0.15.0"
__model_version__ = "0.15.0"
```

If anything else is required to help track down the problem, do not hesitate.

Thanks for your support.

**Comment by adrienball** [comment]
Hey @McFoggy , thanks for reporting this issue.
I managed to replicate it, and filed an [issue](https://github.com/snipsco/rustling-ontology/issues/115) in `rustling-ontology` which is one of our dependency (in Rust), used to parse builtin entities, and which is causing the crash.
We will try to fix it asap, as it is a serious bug.

In the meantime, as a workaround, you can try to use only integer values for the weight entity such as: `"31 kilogrammes"`

**Comment by McFoggy** [comment]
@adrienball thanks for taking that into account. 

Nevertheless I have a follow up question that came to my mind while seeing what you have written in snipsco/rustling-ontology#115:
- in `fr` language, shouldn't "31,2" be interpreted as a single real number (_trente et un virgule deux_ in french or _thirty one dot 2_ in english) instead of 2 distinct integers?

**Comment by adrienball** [comment]
Hey @McFoggy ,

> in fr language, shouldn't "31,2" be interpreted as a single real number (trente et un virgule deux in french or thirty one dot 2 in english) instead of 2 distinct integers?

It is actually the case, `31,2` is correctly interpreted as a decimal number, but what happens under the hood is that there is a data augmentation step in `snips-nlu` which creates many small variations of the dataset utterances in order to improve robustness. One of the variations affects the punctuation, and in your case generates a new sample with "31 2" instead of "31,2".

I hope this is a bit clearer now 😉

**Comment by McFoggy** [comment]
it is thanks a lot for the detailed explanations.

**Comment by adrienball** [comment]
Hey @McFoggy ,
This issue should be fixed with the release `0.16.2`. Could you try and report here ?
Thanks!

**Comment by adrienball** [comment]
I'm closing this issue as it's been 2 weeks since the last update.
Feel free to re-open if you still have troubles.
Cheers
